physionet-django/physionet/settings/base.py

```diff
@@ -75,6 +76,14 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware'
 ]
 
+if ENABLE_SESSION_SECURITY:
+    INSTALLED_APPS += ['session_security']
+    MIDDLEWARE += ['session_security.middleware.SessionSecurityMiddleware']
+    SESSION_EXPIRE_AT_BROWSER_CLOSE = True
```

Comment:
This can be a envvar as well. WDYT?

Reply:
We need it if session security is enabled so I would let it be as it is.